Tests/TestingApps/PurchaseTesterSwiftUI/Shared/StoreKit2ReceiptVerifier.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  StoreKit2ReceiptInspector.swift
+//  PurchaseTester
+//
+//  Created by Andrés Boedo on 3/6/24.
+//
+
+import Foundation
+
+struct StoreKit2ReceiptVerifier {
+
+    func fetchSK2Diagnostics(appConfigID: String, transactionID: String, token: String) async -> String {
```

Comment:
this file could probably be cleaned up to unify the logic with SK1, but it's pretty dead simple anyway, just a single HTTP request